tests/Forc.toml

```diff
@@ -41,6 +41,7 @@ members = [
   "./src/signed_integers/signed_i16_twos_complement",
   "./src/signed_integers/signed_i32_twos_complement",
   "./src/signed_integers/signed_i64_twos_complement",
+  "./src/storage_string",
```

Comment:
libs/Forc.toml has this below the storagemapvec. We should be consistent and pick above / below, similarly in the harness.rs below

Reply:
This follows alphabetical order as does libs/Forc.toml.

Reply:
ah, I've blanked on the 1st portion of the path